lib/pages/chat/events/sending_image_info_widget.dart

```diff
@@ -28,15 +30,17 @@ class SendingImageInfoWidget extends StatelessWidget {
 
   void _onTap(BuildContext context) async {
     if (onTapPreview != null) {
-      await showGeneralDialog(
-        context: context,
-        useRootNavigator: false,
-        barrierDismissible: true,
-        barrierLabel:
-            MaterialLocalizations.of(context).modalBarrierDismissLabel,
-        transitionDuration: const Duration(milliseconds: 200),
-        pageBuilder: (_, animationOne, animationTwo) =>
-            ImageViewer(event, filePath: matrixFile.filePath),
+      Navigator.of(context).push(
```

Comment:
it can affect this issue, can you demo forward case ?
https://github.com/linagora/twake-on-matrix/pull/699